src/services/node/manager.ts

```diff
@@ -0,0 +1,151 @@
+const isProcessRunning = function(processName: string): boolean {
+  const cmd = (() => {
+    switch (require("electron").remote.process.platform) {
+      case "win32":
+        return `tasklist`;
+      case "darwin":
+        return `ps -ax`;
+      case "linux":
+        return `ps -A`;
+      default:
+        return false;
+    }
+  })();
+  const results = require("child_process").execSync(cmd, {
+    windowsHide: true,
+    encoding: "utf-8",
+  });
+  return results.toLowerCase().indexOf(processName.toLowerCase()) > -1;
+};
+
+const killProcess = function(processName: string): void {
+  const cmd = (() => {
+    switch (require("electron").remote.process.platform) {
+      case "win32":
+        return `taskkill /F /IM ${processName}`;
+      case "darwin":
+        return `pkill -9 ${processName}`;
+      case "linux":
+        return `pkill -9 ${processName}`;
+      default:
+        return false;
+    }
+  })();
+  window
+    .require("child_process")
+    .execSync(cmd, { windowsHide: true, encoding: "utf-8" });
+};
+
+export const getNodeDataPath = function(floonet: boolean = false): string {
+  const { remote } = require("electron");
+  const net = !floonet ? "MAINNET" : "FLOONET";
+  return `${remote.app.getPath("home")}/.GrinPP/${net}`;
+};
+
+export const getConfigFilePath = function(floonet: boolean = false): string {
+  return `${getNodeDataPath(floonet)}/server_config.json`;
+};
+
+export const getConfigValue = function(file: string, property: string): number {
+  const data = require("fs").readFileSync(file, "utf8");
+  let settings = JSON.parse(data);
+  if (["MIN_PEERS", "MAX_PEERS"].includes(property)) {
+    if (settings["P2P"][property]) return +settings["P2P"][property];
+    if (property === "MIN_PEERS") return 15;
```

Comment:
Shouldn't this be reading read from defaults.json, instead of just hardcoding?

Reply:
True, actually this is not used anymore since we're using the `defaults.json` file.